atlp-rwanda/eagles-bn-frontend
# update travisCI Coveralls and CodeClimate

What does this PR do?
Setup TravisCI, coveralls, and code climate

Description of Task to be completed?
Setup TravisCI, coveralls, and code climate

How should this be manually tested?
Clone the repository
run npm run test:coveralls to check the test coverage.
Check badges on the Read me page

What are the relevant pivotal tracker stories?
#175226612
